### PR TITLE
Header logger

### PR DESCRIPTION
**Pull request by aryszka**
create a logHeader() filter

### CODE REVIEW

File: docs/reference/filters.md [hunk 1]
```diff
@@ -415,6 +415,13 @@ It logs with INFO level and a unique ID per request:
 - the finishing event of the request
 - any read errors other than EOF
 
+## logHeader
+
+The logHeader filter prints the request line and the header, but not the boddy, to
+stderr. Note that this filter should be used only in diagnositcs setup and with care,
```
Comment (szuecs) [inline docs/reference/filters.md]: diagnositcs -> diagnostics

**Comment by szuecs** [comment]
@aryszka maybe drop the Authorization header from the output or truncate the value?

**Comment by aryszka** [comment]
thought it, but it can be sometimes the goal of the diagnostics, and also there can be other headers containing sensitive data. What do you think about a more drastic safeguard, e.g. a global switch as `-enable-sensitive-diagnostics`? This probably would be the safest way, if we want to keep this filter, but I'm also fine with dropping the PR.

File: docs/reference/filters.md [hunk 2]
```diff
@@ -415,6 +415,13 @@ It logs with INFO level and a unique ID per request:
 - the finishing event of the request
 - any read errors other than EOF
 
+## logHeader
+
+The logHeader filter prints the request line and the header, but not the boddy, to
```
Comment (szuecs) [inline docs/reference/filters.md]: boddy -> body

Comment (aryszka) [reply docs/reference/filters.md]: right, thx!

File: docs/reference/filters.md (hunk 1 above)
Comment (aryszka) [reply docs/reference/filters.md]: thx!

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**